qcommon/build.gradle

```diff
@@ -14,5 +14,7 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.25'
     compile 'org.slf4j:slf4j-simple:1.7.25'
     compile 'com.typesafe:config:1.3.1'
+
+    compile 'javax.xml.bind:jaxb-api:2.2.12' 
```

Comment:
Is this used? Can't see any new imports in the changes that are coming from here.

Reply:
it is java version issue. just make our tool more flexible.

Reply:
I don't understand?

Reply:
From JDK9 there is no JAXB.  Now many users use JDK9+, it is pain to ask users to swipe between java versions. To make our tools can run on java8+, it is better to add them during the build.

Reply:
I think changes for other java versions are out of scope for this PR and should be addressed in their own PR. And in this particular instance, I would presume that we would want to add this dependency at the top level rather than at the individual project level.

Reply:
there is just one line change, it is not necessary to create another PR. It didn't affect anything, just make it more flexible run on JDK8 and above. We may move it to top-level but it will be next PR.

Reply:
I still believe that this is out of scope for this PR. It will result in users who are running on java8 needlessly having to download a redundant jar file that will not be used.
However, there is obviously some resistance to removing this dependency for whatever reason.
At the very least there would need to be some comments in the build file as to why this dependency is being added before I can approve

Reply:
Your comment is incorrect - you don't need the jaxb jar for jdk8